x-pack/legacy/plugins/ml/server/lib/check_license/check_license.test.ts

```diff
@@ -7,15 +7,15 @@
 import expect from '@kbn/expect';
 import sinon from 'sinon';
 import { set } from 'lodash';
-import { XPackInfo } from '../../../../../../../legacy/plugins/xpack_main/server/lib/xpack_info';
-import { checkLicense } from '../check_license';
+import { XPackInfo } from '../../../../xpack_main/server/lib/xpack_info';
+import { checkLicense } from './check_license';
 
 describe('check_license', () => {
   let mockLicenseInfo: XPackInfo;
   beforeEach(() => (mockLicenseInfo = {} as XPackInfo));
 
   describe('license information is undefined', () => {
-    beforeEach(() => (mockLicenseInfo = {} as XPackInfo));
+    beforeEach(() => (mockLicenseInfo = undefined as XPackInfo));
```

Comment:
I had to move the file to run it via jest runner. Mocha doesn't run `* .ts` files, so the test was ignored.